models/amity.py

```diff
@@ -0,0 +1,987 @@
+import collections
+import json
+import os
+import os.path
+import psycopg2
+import psycopg2.extras
+import random
+import re
+import string
+import sys
+
+from operator import itemgetter
+from prettytable import PrettyTable
+from tabulate import tabulate
+
+from models.person import Person, Staff, Fellow
+from models.room import Room, LivingSpace, Office
+
+
+class Amity(object):
+
+    office = []
+    space = []
+    room = []
+
+    staff = []
+    fellow = []
+    person = []
+
+    allocation = []
+
+    CP1_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+    def create_room(self, room_type, room_gender, *rooms_to_add):
+
```

Comment:
comment on function role use, for instance """ This function does the following... """